web/src/jobs/process_article.rs

```diff
@@ -0,0 +1,33 @@
+use crate::AppState;
+use cja::{app_state::AppState as _, jobs::Job};
+use miette::IntoDiagnostic;
+
+use uuid::Uuid;
+
+use db::urls::{store_html_content_in_page_snapshot, Page};
+
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+pub struct ProcessArticle {
+    pub page_id: Uuid,
+}
+
+#[async_trait::async_trait]
+impl Job<AppState> for ProcessArticle {
+    const NAME: &'static str = "process_article::processArticle";
```

Comment:
SuperNit: Lets make this match the struct casing (even though I think I typed this out with you live lol)

```suggestion
    const NAME: &'static str = "process_article::ProcessArticle";
```